arts-university-bournemouth.csl

```diff
@@ -183,8 +183,8 @@
           </group>
         </if>
         <else>
-          <group delimiter=", ">
-            <group delimiter=", ">
+          <group delimiter=". ">
+            <group delimiter=". ">
```

Comment:
Again, this is because i don't know what i'm doing. We just want the full stop after the elements of the full reference, not the in-text citation.